### DISCUSSION TITLE
Will the Flowershow do server side rendering or not?

### DISCUSSION THREAD

**Discussion by saankim** [flowershow/flowershow#146] [General and Help] [answered]
Hi. I'm super excited about this project since I look forward to get SSR and SEO features with obsidian notes on the web.

So, I'm curious will this app provides SSR feature or not.
I read the SEO and meta tag documentation but it's a bit unclear in SSR side.

I hope to see SSR at here! Thx.

**Comment by rufuspollock** [comment, marked as answer]
@saankim right now the core is a standard nextjs app so you can do server side rendering now if you know what you are doing 😄

By default we are doing simple SSG i.e. static rendering so you can deploy anywhere. However, if you want to add server side stuff that is no problem and we do that in some sites we build on top of flowershow.
